JOJO_1104_GAME.py

```diff
@@ -0,0 +1,42 @@
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list, what letters have been guessed so far
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    # FILL IN YOUR CODE HERE...
+    for letter in secret_word:
+        if letter not in letters_guessed:
+            return False
+    return True
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list, what letters have been guessed so far
+    returns: string, comprised of letters and underscores that represents
+      what letters in secret_word have been guessed so far.
+    '''
+    # FILL IN YOUR CODE HERE...
+    String = ""
+    for letter in range(len(secret_word)):
+        if secret_word[letter] in letters_guessed:
+            String += secret_word[letter]
+        else:
+            String += "_"

```

Comment:
When inserting underscores into your string, it's a good idea to add at least a space after each 
one, so it's clear to the user how many unguessed letters are left in the string (compare the 
readability of 
____
with 
_ _ _ _ 
).